client/src/components/icons/DocumentCopyIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export default function DocumentCopyIcon(props: SvgIconProps) {
```

Comment:
[nit] FYI, usually it's better to use named exports rather than default exports, because it makes it easier to refactor and auto-import the components.

But it seems that all the icons are already using default exports, so maybe it's not worth the effort to change this now...